tests/integration/ceph_mgr_test.go

```diff
@@ -123,16 +123,32 @@ func (suite *CephMgrSuite) execute(command []string) (error, string) {
 
 func (suite *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+	var orchStatus map[string]string
 	for timeout := 0; timeout < 30; timeout++ {
 		err, output := suite.execute([]string{"status"})
-		logger.Info("%s", output)
+		logger.Infof("output: %s", output)
 		if err == nil {
 			logger.Info("Rook Toolbox ready to execute commands")
+			// Convert string returned to map
+			output_lines := strings.Split(output, "\n")
```

Comment:
nit: underscores are not commonly a golang naming pattern
```suggestion
			outputLines := strings.Split(output, "\n")
```

Reply:
thx. Sorry ... sorry to forget this things changing from Python to go.... :-)